components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/model/AuthenticationData.java

```diff
@@ -262,11 +268,13 @@ public void addParameters(Map<T1, T2> parameters) {
     }
 
     public Map<T1, T2> getParameters() {
+
         return Collections.unmodifiableMap(parameters);
     }
 
     public T2 getParameter(T1 key) {
+
         return parameters.get(key);
     }
 
-}
+}
```

Comment:
Add new line at the end.

Reply:
Fixed with 9bec2269fed24872017e8e59c1e05f8297fa7518